src/main/java/no/unit/nva/cristin/projects/model/nva/NvaProject.java

```diff
@@ -0,0 +1,131 @@
+package no.unit.nva.cristin.projects.model.nva;
+
+import static no.unit.nva.cristin.projects.JsonPropertyNames.ALTERNATIVE_TITLES;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.CONTEXT;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.CONTRIBUTORS;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.COORDINATING_INSTITUTION;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.END_DATE;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.GRANT;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.ID;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.IDENTIFIER;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.LANGUAGE;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.START_DATE;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.TITLE;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.TYPE;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import java.net.URI;
+import java.time.Instant;
+import java.util.List;
+import java.util.Map;
+import nva.commons.core.JacocoGenerated;
+
+@JacocoGenerated
+@JsonPropertyOrder({CONTEXT, ID, TYPE, IDENTIFIER, TITLE, LANGUAGE, ALTERNATIVE_TITLES, START_DATE, END_DATE,
+    GRANT, COORDINATING_INSTITUTION, CONTRIBUTORS})
+public class NvaProject {
+
+    @JsonProperty(CONTEXT)
+    private String context;
+    private URI id;
+    private String type;
+    @JsonPropertyOrder(alphabetic = true)
+    private List<Map<String, String>> identifier;
+    private String title;
+    private URI language;
+    @JsonPropertyOrder(alphabetic = true)
+    private List<Map<String, String>> alternativeTitles;
+    private Instant startDate;
+    private Instant endDate;
+    //private List<NvaGrant> grant;
```

Comment:
Will be implemented later when we know what to fill it with and also what we are able to get from backend API

Reply:
Create TODO comment and a task in Jira.

Reply:
I think there already is a task in Jira for approvals and fundings/grants. Will make a TODO